test/python/test_instructions.py

```diff
@@ -336,6 +336,7 @@ def test_sign_preattestation(
         snap_path=snap_path / "app_context"
     )
 
+    tezos_navigator.backend.right_click()
```

Comment:
`right_click` without waiting for something afterwards is not safe. At least wait until the screen has changed.

Reply:
What would be right way to implement it as we dont expect any change in the screen when screensaver has turned on. To wake up from screensaver, this right click is implemented. 
Even in check_app_context, we navigate with `            self.navigator.navigate(
                [NavInsID.RIGHT_CLICK] * 2,
                screen_change_before_first_instruction=False
            )`

Reply:
The `screen_change_before_first_instruction=False` option is used to start navigating on the current screen (i.e. the navigator takes the first snapshot directly and does not expect the screen to change before starting to navigate).
And we expect the screen to change because before the `right_click` the screen was empty and after the `right_click` the screen became the first page of the home screen.

Reply:
The `screen_change_before_first_instruction=False` option is used to start navigating on the current screen (i.e. the navigator takes the first snapshot directly and does not expect the screen to change before starting to navigate).
And we expect the screen to change because before the `right_click` the screen was empty and after the `right_click` the screen became the first page of the home screen.

Reply:
Understood. Added a screen check to make sure we have exited the screensaver and are on the Home screen.